### PR TITLE
fix: Updated and added properties for connector

### PR DESCRIPTION
**Pull request by neckron**
fix: Updated and added properties for connector

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<groupId>com.avioconsulting.mule</groupId>
 	<artifactId>mule-health-check-api</artifactId>
 	<packaging>mule-extension</packaging>
-	<version>3.0.1-SNAPSHOT</version>
```
Comment (manikmagar) [inline pom.xml:8]: If its already a snapshot, the version should not be changed. Keep adding to the same snapshot.

File: src/main/resources/org/mule/extensions/smart/connector/module-health check.xml [hunk 2]
```diff
@@ -21,21 +21,21 @@
...
+	<property name="attributes"  type="string" order="1" displayName="Attributes" example="${attributes}" summary="Should always be the attributes predefined var" defaultValue="#[attributes]" use="OPTIONAL"/>
+	<property name="environment" type="string" order="2" displayName="Environment" example="dev" summary="Environment Name" defaultValue="#[p('env')]" use="OPTIONAL"/>
+	<property name="logCategory" type="string" order="3" displayName="Log category"   example="com.avioconsulting.mule.healthcheck" summary="Log category" use="REQUIRED"/>
+	<property name="extSysFlowNames" type="string" order="4" displayName="External systems" example="flow-name-1,flow-name-2" summary="external systems flow names" use="OPTIONAL"/>
+	<property name="healthCheckAppName" type="string" order="5" displayName="Logger app name" example="${app.name}" summary="Environment Name" defaultValue="health-check-api" use="OPTIONAL"/>
+	<property name="healthCheckAppVersion" type="string" order="6" displayName="Logger app version" example="${app.version}" summary="Environment Name" defaultValue="3.0.2" use="OPTIONAL"/>
```
Comment (manikmagar) [inline src/main/resources/org/mule/extensions/smart/connector/module-health check.xml:29]: I think this is supposed to be the name of the app using the health check API. If so why are default values set to health check API name and version? 

Display names "Logger app name/version" and XML element name healthCheckAppName/Version doesn't look consistent. What is the purpose of these attributes?

Comment (adesjardin) [reply src/main/resources/org/mule/extensions/smart/connector/module-health check.xml:29]: Agree with @manikmagar , the default values here should be something like ${app.name} ${api.version} or whatever we are using as the default properties in apps that will consume this.  This should be similar what is the default value for environment above.  

Also the summary for these should not be "Environment Name"

File: pom.xml (hunk 1 above)
Comment (adesjardin) [reply pom.xml:8]: Agreed, keep at 3.0.1 snapshot

File: pom.xml [hunk 3]
```diff
@@ -14,15 +14,27 @@
...
-		<mule.maven.plugin.version>3.8.2</mule.maven.plugin.version>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<maven.compiler.source>1.8</maven.compiler.source>
-		<mule.extensions.maven.plugin.version>1.5.0-20221117</mule.extensions.maven.plugin.version>
-		<avio.logger.version>2.0.1</avio.logger.version>
-		<!-- Publish to Maven Central Properties -->
-		<project.github.repository>avioconsulting/mule-health-check-api</project.github.repository>
-		<repository.url>git@github.com:${project.github.repository}.git</repository.url>
-		<nexus.url>https://oss.sonatype.org</nexus.url>
+        <mule.extensions.maven.plugin.version>1.6.0</mule.extensions.maven.plugin.version>
```
Comment (adesjardin) [inline pom.xml:19]: Please fix the formatting/indentation.  Probably best to make sure the whole file matches format.

**Review by adesjardin — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (neckron) [reply pom.xml]: done

**Review by adesjardin — APPROVED** [review]

**Pull request merged**